offload/test/mapping/skip_transfers.cpp

```diff
@@ -0,0 +1,83 @@
+// clang-format off
+// RUN: %libomptarget-compilexx-generic
+// RUN: env LIBOMPTARGET_INFO=128 %libomptarget-run-generic 2>&1 \
+// RUN: | %fcheck-generic
+
+// REQUIRES: gpu
+// clang-format on
+
+int main() {
+  float DataStack = 0;
+
+// CHECK-NOT: omptarget device 0 info: from(unknown)[4] is not used and will not
```

Comment:
Check, if possible, the message that says it's copied, not the missing message.

Reply:
Done